### ISSUE TITLE
Implement continuable-mocha

### ISSUE THREAD

**Issue by dchambers** [BladeRunnerJS/brjs#1593]
Implement [continuable-mocha](https://github.com/BladeRunnerJS/continuable-mocha).

**Comment by dchambers** [comment]
@andyberry88, do you mind if I finesse the 'continuable-mocha' `README.md` a little?

**Comment by thecapdan** [comment]
issue raised in [continuable-mocha](https://github.com/BladeRunnerJS/continuable-mocha) to ensure it is backwardly compatible

**Comment by thecapdan** [comment]
also @andyberry88 the test fail when i run them with karma:  TypeError: thisSuiteIgnoredTests.forEach is not a function. same for chrome and firefox.

**Comment by dchambers** [comment]
@thecapdan, isn't this now done? I thought that only the verifier fixtures (#1595) had any outstanding work left?

**Comment by thecapdan** [comment]
@dchambers the tests pass when i run them in node, but not when i run them in the browser with karma (firefox/chrome on windows). this needs to be addressed before we can close this

**Comment by thecapdan** [comment]
The forEach issue was because of an old browser version that didn't support foreach on an object.

But there is another issue where we go into an infinite loop when we continue from a test in another file... It's not the most important use case, but I'll have a quick look to see if it can be fixed

**Comment by thecapdan** [comment]
The bug is worse than I originally thought. The bug is whenever we have two test files using continuesFrom, we go into an infinite loop since it creates a global version of describe which ends up calling itself over and over.

Ensuring only the first test suite imports continuesFrom  fixes the issue, but that isn't a good solution. Needs a closer look

**Comment by thecapdan** [comment]
These issues are now fixed + we now have a travis build with link on the main page.

Done
